#! /bin/sh /usr/share/dpatch/dpatch-run
## 010disabling_update_note.dpatch by Giuseppe Iuculano <giuseppe@iuculano.it>
##
## All lines beginning with `## DP:' are a description of the patch.
## DP: Disabled the the "please update" warning, thanks to Hans Spaans and Rolf Leggewie (Closes: #506685)

@DPATCH@
diff -urNad wordpress~/wp-admin/includes/update.php wordpress/wp-admin/includes/update.php
--- wordpress~/wp-admin/includes/update.php	2009-08-10 17:25:44.000000000 +0200
+++ wordpress/wp-admin/includes/update.php	2009-08-11 16:10:55.000000000 +0200
@@ -30,6 +30,8 @@
  * @return array Array of the update objects
  */
 function get_core_updates( $options = array() ) {
+	if ( !constant('WP_CORE_UPDATE') )
+		return false;
 	$options = array_merge( array('available' => true, 'dismissed' => false ), $options );
 	$dismissed = get_option( 'dismissed_update_core' );
 	if ( !is_array( $dismissed ) ) $dismissed = array();
